vector.yaml

```
@@ -0,0 +1,31 @@
+sources:
+  docker_logs:
+    type: docker_logs
+    exclude_containers: 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)